linode/manager
# DBAAS1-86: Database Details Settings

## Description

Adds the database settings tab to the database details page. This includes three sections:

- Add Access Controls
- Reset Root Password
- Delete Database Cluster

Resetting the root password and deleting the database will trigger confirmation dialogs for each.

## How to test

1. Navigate to the database details page.
2. Click the Settings tab
3. Ensure there are sections for Add Access Controls, Reset Root Password, Delete Database Cluster
4. Ensure the proper modals appear for resetting the root password and deleting the cluster
5. Ensure the proper HTTP calls are made to complete the tasks for each section.

`yarn run test DatabaseSettings`
